website/static/js/contribAdder.js

```diff
@@ -60,10 +79,17 @@ var AddContributorViewModel = oop.extend(Paginator, {
         self.results = ko.observableArray([]);
         self.contributors = ko.observableArray([]);
         self.selection = ko.observableArray();
+        self.contributorIDsToAdd = ko.computed(function() {
+            var contributorIDs = [];
```

Comment:
Minor: when building an Array based on values from another Array,  use `.map` rather than a for loop. Also, this computed can be a `pureComputed`.